source/fundamentals/crud/read-operations/watch.txt

```diff
@@ -118,53 +113,90 @@ new delete operations:
 
    pipeline := bson.D{{"$match", bson.D{{"operationType", "delete"}}}}
    changeStream, err := db.Watch(context.TODO(), mongo.Pipeline{pipeline})
-   if err != nil {
-      panic(err)
-   }
-   defer changeStream.Close(context.TODO())
-
-   for changeStream.Next(context.TODO()) {
-      fmt.Println(changeStream.Current)
-   }
-
-Deleting the document with the ``title`` value of "Advanced Screenwriting"
-in a separate program or shell results in the following change-stream event:
-
-.. code-block:: go
-   :copyable: false
-
-   {"_id": {"_data": "..."},"operationType": "delete","clusterTime":
-   {"$timestamp":{"t":"...","i":"..."}},"ns": {"db": "db","coll": "courses"},
-   "documentKey": {"_id": {"$oid":"..."}}}
 
 .. note::
 
    The ``Watch()`` method was called on the ``db`` database, so the code outputs
    new delete operations in any collection within this database.
 
-Modify the Behavior of Watch()
-------------------------------
+Modify the Behavior of ``Watch()``
+----------------------------------
 
-Use an opts parameter to modify the behavior of the ``Watch()`` method.
+Use options to modify the behavior of the ``Watch()`` method.
 
-You can specify the following options in the opts parameter:
+You can specify the following options on the ``Watch()`` method:
 
 - ``ResumeAfter``
 - ``StartAfter``
 - ``FullDocument``
+- ``FullDocumentBeforeChange``
 - ``BatchSize``
 - ``MaxAwaitTime``
 - ``Collation``
 - ``StartAtOperationTime``
+- ``Comment``
+- ``ShowExpandedEvents``
+- ``StartAtOperationTime``
+- ``Custom``
+- ``CustomPipeline``
+
+For more information on these options, visit the
+:manual:`MongoDB Server manual </reference/method/db.collection.watch/>`.
 
-For more information on these fields, visit the
-:manual:`MongoDB manual </reference/method/Mongo.watch/#mongodb-method-Mongo.watch>`.
+Pre- and Post-Images
+~~~~~~~~~~~~~~~~~~~~
+
+When you perform any CRUD operation on a collection, by default, the corresponding
+change event document contains only the delta of the fields modified
+by the operation. To access the pre-image (the document before a
+change) or the post-image (the document after a change) of a document,
+set the ``FullDocument`` and ``FullDocumentBeforeChange`` fields in the
+options for ``Watch()``.
+
+.. note::
+
+   There is no pre-image for an inserted document and no post-image for
+   a deleted document.
+
+If you want to access the entire most current
+majority-committed version of the changed document, specify the
+``FullDocument`` options parameter using the setter method.
+If you enable this feature, ``FullDocument`` can take the
+following values:
+
+- ``UpdateLookup``: The change event document includes a delta
```

Comment:
```suggestion
- ``UpdateLookup``: The change event document includes the delta
```

(not just, like, any arbitrary old delta. _The_ delta.)

Reply:
moved reference from the delta here as the introductory text now explains that these options are in addition to the delta, which is given by default